bindings/pyroot/pythonizations/python/ROOT/pythonization/_roofit/__init__.py

```diff
@@ -193,13 +193,22 @@ def pythonize_roofit_class(klass, name):
         if hasattr(klass, func_name):
             # new name for original function
             func_name_orig = make_func_name_orig(func_name)
+            func_orig = getattr(klass, func_name)
+            func_new = getattr(python_klass, func_name)
 
-            setattr(klass, func_name_orig, getattr(klass, func_name))
+            if func_new.__doc__ is None:
+                func_new.__doc__ = func_orig.__doc__
+            else:
+                func_new.__doc__ += func_orig.__doc__
```

Comment:
```suggestion
            import inspect

            if func_new.__doc__ is None:    
                func_new.__doc__ = func_orig.__doc__    
            elif not func_orig.__doc__ is None:    
                python_docstring = func_new.__doc__    
                func_new.__doc__ = "Documentation of pythonized function\n"    
                func_new.__doc__ += "====================================\n\n"    
                func_new.__doc__ += inspect.cleandoc(python_docstring) + "\n\n"    
                func_new.__doc__ += "Documentation of original cppyy.CPPOverload object\n"    
                func_new.__doc__ += "==================================================\n\n"    
                func_new.__doc__ += func_orig.__doc__ 
```

I suggest to make this concatenation a bit nicer, structuring the doc strings in two labeled sections. It would also be good to use the `inspect.cleandoc` function to remove indentation from the docstring.